tests/test_fakes.py

```diff
@@ -884,7 +884,6 @@ def test_show_objects(dcur: snowflake.connector.cursor.SnowflakeCursor):
             "schema_name": "SCHEMA1",
         },
     ]
-    assert dcur.fetchall() == objects
```

Comment:
this should still work i think?

Reply:
It's just that i'm testing both terse and non-terse, so I'd have to mutate the original structure to add the "comment
" column. i figured it was less relevant than just showing the column now exists